project-base/storefront/pages/social-login.tsx

```diff
@@ -0,0 +1,40 @@
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+import { getStringFromUrlQuery } from 'utils/parsing/getStringFromUrlQuery';
+import { getServerSidePropsWrapper } from 'utils/serverSide/getServerSidePropsWrapper';
+import { initServerSideProps, ServerSidePropsType } from 'utils/serverSide/initServerSideProps';
+import { showErrorMessage } from 'utils/toasts/showErrorMessage';
+
+const PaymentStatusNotifyPage: FC<ServerSidePropsType> = () => {
```

Comment:
I think you forgot to change the component name :)

Reply:
Thanks